src/components/rangeSlider/RangeSliderWithChart.tsx

```diff
@@ -115,10 +129,14 @@ const RangeSliderWithChart: React.FC<RangeSliderWithChartProps> = ({
         step={1}
         min={0}
         max={scale.length}
-        onChange={([newMinIndex, newMaxIndex]) => {
-          onSliderChange(toMinMax(newMinIndex, newMaxIndex, selection));
+        onChange={(newValues) => handleChange(newValues, onSliderChange)}
+        onChangeEnd={(newValues) => {
+          const callback = (event: ChangeCallback) => {
+            onSliderRelease(event);
+            setStartRange(newValues);
```

Comment:
this prevents an infinite loop that existed before if you used the arrows to change the filter values